picard/webservice/__init__.py

```diff
@@ -20,31 +21,30 @@
 
 
 """
-Asynchronous XML web service.
+Asynchronous web service.
 """
 
 import sys
-import re
 import time
 import os.path
 import platform
 import math
 from collections import deque, defaultdict
 from functools import partial
 from PyQt5 import QtCore, QtNetwork
-from PyQt5.QtCore import QUrl, QXmlStreamReader, QStandardPaths, QUrlQuery
+from PyQt5.QtCore import QUrl, QStandardPaths, QUrlQuery
 from picard import (PICARD_APP_NAME,
                     PICARD_ORG_NAME,
                     PICARD_VERSION_STR,
                     config,
                     log)
-from picard.const import (ACOUSTID_KEY,
-                          ACOUSTID_HOST,
+from picard.const import (ACOUSTID_HOST,
```

Comment:
Those are now only used for request delays (`REQUEST_DELAY` dict below), i think it can be moved to API helpers as well. Not sure exactly how though ;)